src/components/Header/index.tsx

```diff
@@ -312,13 +296,17 @@ function Header() {
             {t('liquidity')}
             {networkWithoutSWPR && <HeaderLinkBadge label="NOT&nbsp;AVAILABLE" />}
           </HeaderLink>
+          <HeaderLink data-testid="liquidity-v3-nav-link" id="liquidity-v3-nav-link" href={LIQUIDITY_V3_LINK}>
+            {t('liquidityV3')}
+            <NewBadge>NEW</NewBadge>
+            {networkWithoutSWPR && <HeaderLinkBadge label="NOT&nbsp;AVAILABLE" />}
```

Comment:
I think `networkWithoutSWPR ` does not make sense here. We are not showing "Not available" on Ethereum. Also, I think we shouldn't show "Not available" on a new product, we want them to click.

Reply:
Ok! Removing it now!